check/controls.go

```diff
@@ -23,10 +23,11 @@ import (
 
 // Controls holds all controls to check for master nodes.
 type Controls struct {
-	ID     string `yaml:"id"`
-	Text   string
-	Type   NodeType
-	Groups []*Group
+	ID      string `yaml:"id"`
+	Version string
```

Comment:
Do we use this Version field anywhere? I couldn't see it but perhaps I missed it.